### ISSUE TITLE
Running out of memory for PersistentVolumeClaim

### ISSUE THREAD

**Issue by david-sabata** [derailed/popeye#38]
<img src="https://raw.githubusercontent.com/derailed/popeye/master/assets/popeye.png" align="right" width="100" height="auto"/>

<br/>
<br/>
<br/>

**Describe the bug**
Popeye runs out of memory on one of my clusters (not all of them) if `pvc` section is included. Running report with all the other sections works fine.
The cluster runs in GKE. 
**Please let me know what additional info I can provide to help you debug this** :wink: 

**To Reproduce**
1. Run `popeye -s pvc`
2. See error

**Expected behavior**
Seeing the report :grinning: 

**Versions (please complete the following information):**
 - OS: Ubuntu 19.04, 16GB of RAM
 - Popeye 0.3.11
 - K8s 1.12.7-gke.10

**Full stack trace**
```
fatal error: runtime: out of memory

runtime stack:
runtime.throw(0x14d528d, 0x16)
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/panic.go:617 +0x72
runtime.sysMap(0xcaa0000000, 0x2b0000000, 0x2294d78)
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/mem_linux.go:170 +0xc7
runtime.(*mheap).sysAlloc(0x227c520, 0x2acf5c000, 0x227c530, 0x1567ae)
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/malloc.go:633 +0x1cd
runtime.(*mheap).grow(0x227c520, 0x1567ae, 0x0)
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/mheap.go:1222 +0x42
runtime.(*mheap).allocSpanLocked(0x227c520, 0x1567ae, 0x2294d88, 0x7f6b9f45d760)
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/mheap.go:1150 +0x37f
runtime.(*mheap).alloc_m(0x227c520, 0x1567ae, 0x7f6b3f3e0100, 0x7f6b3f3e6940)
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/mheap.go:977 +0xc2
runtime.(*mheap).alloc.func1()
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/mheap.go:1048 +0x4c
runtime.(*mheap).alloc(0x227c520, 0x1567ae, 0xc000010100, 0x7f6b3f3e68b0)
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/mheap.go:1047 +0x8a
runtime.largeAlloc(0x2acf5c000, 0x450001, 0x7f6b3f3e68b0)
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/malloc.go:1055 +0x99
runtime.mallocgc.func1()
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/malloc.go:950 +0x46
runtime.systemstack(0xc00004acc0)
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/asm_amd64.s:351 +0x66
runtime.mstart()
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/proc.go:1153

goroutine 1 [running]:
runtime.systemstack_switch()
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/asm_amd64.s:311 fp=0xc000483748 sp=0xc000483740 pc=0x4561c0
runtime.mallocgc(0x2acf5c000, 0x12c5a80, 0x1, 0x1)
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/malloc.go:949 +0x872 fp=0xc0004837e8 sp=0xc000483748 pc=0x40bcb2
runtime.growslice(0x12c5a80, 0xc87c000000, 0x223f7c00, 0x223f7c00, 0x223f7c01, 0xc87c000000, 0x1b65fc00, 0x223f7c00)
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/slice.go:181 +0x1e6 fp=0xc000483850 sp=0xc0004837e8 pc=0x441516
github.com/derailed/popeye/internal/report.formatLine(0xc00049c070, 0x63, 0x1, 0x60, 0xc0001ac768, 0xc0001fa9c0)
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/internal/report/writer.go:197 +0x2b7 fp=0xc000483930 sp=0xc000483850 pc=0x109a4b7
github.com/derailed/popeye/internal/report.(*Sanitizer).write(0xc0003e0380, 0x2, 0x1, 0xc00049c070, 0x63)
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/internal/report/writer.go:144 +0x188 fp=0xc000483a38 sp=0xc000483930 pc=0x1099ab8
github.com/derailed/popeye/internal/report.(*Sanitizer).Print(...)
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/internal/report/writer.go:128
github.com/derailed/popeye/internal/report.(*Builder).PrintReport(0xc0002313e0, 0x0, 0xc0003e0380)
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/internal/report/builder.go:188 +0x616 fp=0xc000483bc8 sp=0xc000483a38 pc=0x10974c6
github.com/derailed/popeye/pkg.(*Popeye).dump(0xc00022bdc0, 0xc000231301)
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/pkg/popeye.go:92 +0x2ba fp=0xc000483ce0 sp=0xc000483bc8 pc=0x10b814a
github.com/derailed/popeye/pkg.(*Popeye).Sanitize(0xc00022bdc0)
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/pkg/popeye.go:100 +0x3e fp=0xc000483d00 sp=0xc000483ce0 pc=0x10b861e
github.com/derailed/popeye/cmd.doIt(0x2267740, 0xc00037f1f0, 0x0, 0x7)
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/cmd/root.go:62 +0x11b fp=0xc000483d78 sp=0xc000483d00 pc=0x10b953b
github.com/spf13/cobra.(*Command).execute(0x2267740, 0xc00003a090, 0x7, 0x7, 0x2267740, 0xc00003a090)
	/Users/fernand/go_wk/derailed/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2ae fp=0xc000483e60 sp=0xc000483d78 pc=0xfdca9e
github.com/spf13/cobra.(*Command).ExecuteC(0x2267740, 0x0, 0x0, 0xc00037df88)
	/Users/fernand/go_wk/derailed/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec fp=0xc000483f30 sp=0xc000483e60 pc=0xfdd6ec
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/fernand/go_wk/derailed/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
github.com/derailed/popeye/cmd.Execute()
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/cmd/root.go:38 +0x32 fp=0xc000483f88 sp=0xc000483f30 pc=0x10b9392
main.main()
	/Users/fernand/go_wk/derailed/src/github.com/derailed/popeye/main.go:29 +0x20 fp=0xc000483f98 sp=0xc000483f88 pc=0x11f3020
runtime.main()
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/proc.go:200 +0x20c fp=0xc000483fe0 sp=0xc000483f98 pc=0x42d82c
runtime.goexit()
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/asm_amd64.s:1337 +0x1 fp=0xc000483fe8 sp=0xc000483fe0 pc=0x458111

goroutine 5 [chan receive]:
k8s.io/klog.(*loggingT).flushDaemon(0x2275aa0)
	/Users/fernand/go_wk/derailed/pkg/mod/k8s.io/klog@v0.1.0/klog.go:941 +0x8b
created by k8s.io/klog.init.0
	/Users/fernand/go_wk/derailed/pkg/mod/k8s.io/klog@v0.1.0/klog.go:403 +0x6c

goroutine 7 [select]:
go.opencensus.io/stats/view.(*worker).start(0xc00022b840)
	/Users/fernand/go_wk/derailed/pkg/mod/go.opencensus.io@v0.19.0/stats/view/worker.go:147 +0xdd
created by go.opencensus.io/stats/view.init.0
	/Users/fernand/go_wk/derailed/pkg/mod/go.opencensus.io@v0.19.0/stats/view/worker.go:29 +0x57

goroutine 36 [IO wait, 2 minutes]:
internal/poll.runtime_pollWait(0x7f6b9d1e3f08, 0x72, 0xffffffffffffffff)
	/usr/local/Cellar/go/1.12.5/libexec/src/runtime/netpoll.go:182 +0x56
internal/poll.(*pollDesc).wait(0xc000421b98, 0x72, 0x14600, 0x146bc, 0xffffffffffffffff)
	/usr/local/Cellar/go/1.12.5/libexec/src/internal/poll/fd_poll_runtime.go:87 +0x9b
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/Cellar/go/1.12.5/libexec/src/internal/poll/fd_poll_runtime.go:92
internal/poll.(*FD).Read(0xc000421b80, 0xc000382000, 0x146bc, 0x146bc, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.12.5/libexec/src/internal/poll/fd_unix.go:169 +0x19b
net.(*netFD).Read(0xc000421b80, 0xc000382000, 0x146bc, 0x146bc, 0x203000, 0x0, 0x135b2)
	/usr/local/Cellar/go/1.12.5/libexec/src/net/fd_unix.go:202 +0x4f
net.(*conn).Read(0xc000454000, 0xc000382000, 0x146bc, 0x146bc, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.12.5/libexec/src/net/net.go:177 +0x69
crypto/tls.(*atLeastReader).Read(0xc00040c040, 0xc000382000, 0x146bc, 0x146bc, 0x100, 0x7f6b9d43efff, 0xc00047f9e0)
	/usr/local/Cellar/go/1.12.5/libexec/src/crypto/tls/conn.go:761 +0x60
bytes.(*Buffer).ReadFrom(0xc00045c258, 0x16a1320, 0xc00040c040, 0x409835, 0x134a000, 0x146c860)
	/usr/local/Cellar/go/1.12.5/libexec/src/bytes/buffer.go:207 +0xbd
crypto/tls.(*Conn).readFromUntil(0xc00045c000, 0x16a3320, 0xc000454000, 0x5, 0xc000454000, 0x9)
	/usr/local/Cellar/go/1.12.5/libexec/src/crypto/tls/conn.go:783 +0xf8
crypto/tls.(*Conn).readRecordOrCCS(0xc00045c000, 0x1559500, 0xc00045c138, 0xc00047fd58)
	/usr/local/Cellar/go/1.12.5/libexec/src/crypto/tls/conn.go:590 +0x125
crypto/tls.(*Conn).readRecord(...)
	/usr/local/Cellar/go/1.12.5/libexec/src/crypto/tls/conn.go:558
crypto/tls.(*Conn).Read(0xc00045c000, 0xc0004b9000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.12.5/libexec/src/crypto/tls/conn.go:1236 +0x137
bufio.(*Reader).Read(0xc000450b40, 0xc0004ba038, 0x9, 0x9, 0x406234, 0xc000230300, 0xc00047fd58)
	/usr/local/Cellar/go/1.12.5/libexec/src/bufio/bufio.go:223 +0x23e
io.ReadAtLeast(0x16a11a0, 0xc000450b40, 0xc0004ba038, 0x9, 0x9, 0x9, 0x16a14c0, 0xc000252040, 0xc00008a060)
	/usr/local/Cellar/go/1.12.5/libexec/src/io/io.go:310 +0x88
io.ReadFull(...)
	/usr/local/Cellar/go/1.12.5/libexec/src/io/io.go:329
golang.org/x/net/http2.readFrameHeader(0xc0004ba038, 0x9, 0x9, 0x16a11a0, 0xc000450b40, 0x0, 0x0, 0xc0000b2390, 0x0)
	/Users/fernand/go_wk/derailed/pkg/mod/golang.org/x/net@v0.0.0-20190311183353-d8887717615a/http2/frame.go:237 +0x88
golang.org/x/net/http2.(*Framer).ReadFrame(0xc0004ba000, 0xc0000b2390, 0x0, 0x0, 0x0)
	/Users/fernand/go_wk/derailed/pkg/mod/golang.org/x/net@v0.0.0-20190311183353-d8887717615a/http2/frame.go:492 +0xa1
golang.org/x/net/http2.(*clientConnReadLoop).run(0xc00047ffb8, 0x15582d8, 0xc0004687b8)
	/Users/fernand/go_wk/derailed/pkg/mod/golang.org/x/net@v0.0.0-20190311183353-d8887717615a/http2/transport.go:1679 +0x8d
golang.org/x/net/http2.(*ClientConn).readLoop(0xc00045a300)
	/Users/fernand/go_wk/derailed/pkg/mod/golang.org/x/net@v0.0.0-20190311183353-d8887717615a/http2/transport.go:1607 +0x76
created by golang.org/x/net/http2.(*Transport).newClientConn
	/Users/fernand/go_wk/derailed/pkg/mod/golang.org/x/net@v0.0.0-20190311183353-d8887717615a/http2/transport.go:670 +0x637
```

**Comment by derailed** [comment]
@david-sabata Thank you for this great report! Fixed 0.3.12.

**Comment by david-sabata** [comment]
Thanks for the quick fix and an awesome tool!
